### ISSUE TITLE
Cell hashing experiment for TCR

### ISSUE THREAD

**Issue by clairebio27** [Hoohm/CITE-seq-Count#128]
Our objective is  to test TCR Ab Oligos to define starting cell type, and also to Test the ability to pool samples with TCR Ab Oligos. 

We are using 10X for single cell  analysis and illumina for sequencing. We have used 9 different HTOs and have 9 samples . 

This is something I am trying for the first time and therefore I am reaching out to see a workflow for the analysis to meet the objectives.
I have 9 fastqs (R1,R2 and I for each sample) for ADT(HTOs),9 fastqs for TCR from illumina , 
What I have done so far is:
1. sample wise CITE-Seq providing tag.csv
Questions:
Do I need to pool all samples and do the UMI count (CITE-seq)
I want to do the  demultiplexing of  hashed samples and later annotate  to clonotypes?
What downstream workflow I should use ?
Any help in this regard would be greatly appreciated. Please let me know if require any more details

**Comment by Hoohm** [comment]
Hello @clairebio27,
quite a complex setup you got there.

I'm not sure I understood everything, but I would use the link between the HTO and the barcodes to decipher which TCR comes from which cells, then use this as annotation for your TCR analysis.

Regarding the HTO, do you have one per fastqs? Or do you have 9 starting cell types mixed across all the 9 HTO fastqs?

**Comment by clairebio27** [comment]
Hi Patrick,

Thank you for the email.

I  have cell types mixed across all (mostly more than one HTO) HTO fastqs.

I have used a naive approach to analyse this data.
1. I have processed the fastqs and retrieved the cell barcodes and feature
barcodes (HTO tags) by parsing R1 and R2 fastqs using python pandas.
2.  Sepraelty done a VDJ assembly (using cell ranger) for each fastqs.
3. Then after merging the retrieved  HTO tags and cell barcodes (from HTO
fastqs and TCR), I tried to see the counts/expression of each HTOs from
each sample and its corresponding annotation.

So in short, I haven't used  cell rager count or CITE seq or Seurat for the
analysis.

Could these results be wrong?
What could be the difference in results  if I do the analysis this way and
HTOdemux using Seurat? Am I going the wrong direction?

Claire

**Comment by Hoohm** [comment]
Hello @clairebio27,
I'm sorry but I'm having a hard time following your main question. The design of the experiment is not clear to me and I would not want to tell you something wrong.

In 1, I notice you have not retrieved the UMI. This means you won't have any deduplication information. SO your data will be "biased" by potential different duplication rates.

If the design is right, what you miss out by doing everything yourself is cell barcode correction, umi correction, mapping errors that are both being done in cell ranger and CSC.
